src/components/RegistrationPanel.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useRef } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { FaXmark } from 'react-icons/fa6';
+
+import { deactivateRegistrationPanel } from '../features/users/usersSlice';
+
+import LoginForm from '../features/users/LoginForm';
+import SignUpForm from '../features/users/SignUpForm';
+
+import '../styles/components/RegistrationPanel.scss';
+
+const RegistrationPanel = () => {
```

Comment:
- [ ] I cannot create a new user, no matter how many times I try:

![image](https://github.com/ITurres/tesla-booking-front-end/assets/57421823/d88aed2e-afce-4fdb-bbe8-0922399e3dd4)

After some investigating, I could see that this entire frontend app depends on a .env file that has not been included, and the project has calls for it in many files:

![image](https://github.com/ITurres/tesla-booking-front-end/assets/57421823/fd6291cc-67e9-484d-abef-23d2eb4bf316)


Therefore, kindly include on your readme specific instructions about how to generate such .env file with the correct properties. 

That way, all your users will be able to run and setup the frontend app correctly. 👍 

_note: That means we couldn't test the rest of the frontend app's functionality._

Reply:
### Hi @Meltrust! Thank you for reviewing this PR 🎯

Firstly, I'd like to mention that this PR has already been merged into `main`, and the assessment has been requested. Could you please guide us on where to make the required changes - in `development` or `main`?

Secondly, the `.env` file is intentionally ignored by git due to its sensitive content, such as API endpoints. We've securely set these environmental variables in the repo's settings under the _*Secrets and variables*_ for the `development` environment. These variables will be utilized by the `deploy.yml` file and gh-pages during deployment.

Nevertheless, I'll create a `.env_sample` file to demonstrate how to include API keys.

Once again, thank you for your assistance. _**Please advise whether we should push this into `main` or `development`, considering that the assessment has already been requested.**_

I appreciate your prompt response, and I'll be awaiting further instructions.

Reply:
Hello @ITurres, and thanks for your inquiry.  The reviewed branch was `development`, as requested.  

I understand that you have set this project to be viewed on the cloud (good job on that).  However, for this particular project, we are required to review it locally and I was greeted with a set of secrets/environmental variables that would imply refactoring several parts of the project, in order to make it talk to the local server.

To respond to your question: normally, for the final evaluation you merge from `development` to `main`, which you already did.

Since this is a special review designed to improve your chances on the final test, if you want to make no further changes to the project, you can ignore my suggestions and go ahead to final evaluation.

I wish you good luck with the final evaluation and congratulations on finishing Microverse 👏

Reply:
Dear @Meltrust,

Thank you for your prompt response! I appreciate your valuable assistance. I am proceeding with the creation of the mentioned file and updating the documentation. These changes will be pushed into the `main` branch.

Once again, I want to express my gratitude for the time and attention you've dedicated to this matter. 🥂